frontend/src/components/NewIssueModal/NewIssueModal.tsx

```diff
@@ -242,6 +242,7 @@ const NewIssueModal: React.FC<React.PropsWithChildren<NewIssueModalProps>> = ({
 							name={form.names.issueDescription}
 							label="Issue Description"
 							placeholder="Hey, check this out!"
+							/* @ts-ignore */
 							as="textarea"
```

Comment:
I couldn't figure out how to fix this issue whereby `InputProps` is already declared as type "input"
https://github.com/highlight/highlight/blob/c1726dc404b3bbb4d5dfb557807bdb613f98b399/packages/ui/src/components/Form/Form.tsx#L66-L70

I tried a couple approaches (including making a distinct `Form.Textarea` component) and struggled with the types.

Will leave a ts-ignore in for now as the larger issue of incomplete types for our form components is more important to fix.